en-US/Samples/CylonLEDNode.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: sample
-title: Cylon servo controller
-description: Use Cylon to blink an LED on an Arduino from a Windows 10 IoT Core device
-keywords: Windows 10 IoT Core, wiring, arduino
-permalink: /en-US/Samples/CylonServoWiring.htm
+title: Cylon LED controller
+description: Blink an Arduino LED using Cylon in a Node.js console app
+keywords: Windows 10 IoT Core, arduino
+permalink: /en-US/Samples/CylonLEDNode.htm
```

Comment:
If changing a file name/location and permalink, we require that we put in a client side redirect for the old file - will you please add back in a CylonServoWiring that redirects to this CylonLEDNode?  info on that can be found in the contribute docs, or copy the way we do in en-US/win10/Docs